COPYRIGHT

```diff
@@ -6,7 +6,7 @@ which must be included in the prologue of the code and in all source listings
 of the code.
 
 Copyright Notice
- + 2002 University of Chicago
+2020, Argonne National Laboratory
```

Comment:
We are expected to update this every year, right?

Some projects use a year range, as some of the code are old and some are new. Should we use `2002 - 2020`?

Reply:
We could use a range.  I'll add that.  I need to go back to CVS and find the earliest commit.